includes/CMB2_Base.php

```diff
@@ -379,6 +379,43 @@ public static function maybe_hook( $val, $hook_name, $hook_function ) {
 	}
 
 	/**
+	 * Arguments for custom wp_kses sanitization/validation calls.
+	 *
+	 * @since  2.2.4
+	 * @param  string $type The field type being escaped.
+	 * @return array        An array of wp_kses HTML parameters allowed.
+	 */
+	public function kses_args( $type ) {
+		$params = array();
+
+		switch ( $type ) {
+			case 'group_wrap_attributes':
+				$params = array(
+					'class' => array(),
+					'style' => array(),
+				);
+				break;
+			case 'remove_disabled':
+				$params = array(
+					'disabled' => array(),
```

Comment:
Partially because I see this function expanding to various wp_kses arguments for different things we need to sanitize throughout the code, so ideally there would be one place for all the things.